substrate/frame/support/procedural/src/pallet/expand/tt_default_parts.rs

```diff
@@ -79,6 +81,56 @@ pub fn expand_tt_default_parts(def: &mut Def) -> proc_macro2::TokenStream {
 		.any(|c| matches!(c.composite_keyword, CompositeKeyword::SlashReason(_)))
 		.then_some(quote::quote!(SlashReason,));
 
+	let call_part_v2 = def.call.as_ref().map(|_| quote::quote!(+ Call));
```

Comment:
These seem like extra code that we generate, only because we are trying to support both for now. Would be good to measure if the compilation time has increased by much here.

Reply:
Yeah, I didn't see any noticeable difference but will confirm.